frontend/www/js/omegaup/components/problem/Mine.vue

```diff
@@ -110,7 +110,13 @@
                   <div v-if="problem.tags.length" class="tags-badges">
                     <a
                       v-for="tag in problem.tags"
-                      :class="`badge custom-badge custom-badge-${tag.source} m-1 p-2`"
+                      :class="`badge custom-badge custom-badge-${
+                        tag.source.includes('quality') ? 'owner' : tag.source
+                      } ${
+                        tag.name.includes('problemLevel')
+                          ? 'custom-badge-quality'
+                          : ''
+                      } m-1 p-2`"
```

Comment:
se puede usar la [sintaxis de arreglo](https://vuejs.org/v2/guide/class-and-style.html#Array-Syntax)?

```suggestion
                      class="badge custom-badge m-1 p-2"
                      :class="[
                        {
                          'custom-badge-quality': tag.name.includes('problemLevel'),
                        },
                        `custom-badge-${
                          tag.source.includes('quality') ? 'owner' : tag.source
                        }`,
                      ]"
```